wallet-core/wallet-core-usage.md

```diff
@@ -314,7 +309,7 @@ let signingInput = BinanceSigningInput.with {
     }
 }
 
-let data = BinanceSigner.sign(input: signingInput)
+let data: BinanceSigningOutput = AnySigner.sign(input: signingInput, coin: .binance)
```

Comment:
```suggestion
let data: BinanceSigningOutput = AnySigner.sign(input: input, coin: .binance)
```

should it be consistent for both? `input` and `output`?

Reply:
done